### PR TITLE
🔥 CONTRAST-32208 Removing steps which instruct user how to add integration tests (maven)

### PR DESCRIPTION
**Pull request by gilday**
These edits reflect changes I'm making to @rszewczyk's draft prior to transferring this content over to @kaitlynmortimer to copy edit and include in Open Docs. The plan is to migrate the `how-to.md` file to Open Docs and keep the code examples here on Github (the article refers to the code examples). The plan for releasing this content is

1. Prepare final draft on Github (this PR). Merge to a branch (CONTRAST-31003)
1. Migrate `how-to.md` to Open Docs and turn over to @kaitlynmortimer for copy edits
1. Wait for Contrast Java agent to be released to Maven Central
1. Release the Open Docs article and merge this content to master effectively publishing this article

Prior to these changes, this article showed users how to add integration tests to a project using the `maven-cargo-plugin` and subsequently how to configure the `maven-cargo-plugin` to include Contrast. This change re-focuses the article on just adding Contrast to an existing `maven-cargo-plugin` configuration.

@rszewczyk and I want to move the "how to add integration tests that work with Contrast" content to  to an [explanation article](https://www.divio.com/blog/documentation/) that is goes into the details of automated testing + Contrast, and we want to keep this how-to focused on Contrast + maven-cargo-plugin.

Other changes:

* Updated artifactId java-agent --> contrast-agent
* Removed the jar renaming exercise that will be obviated by the changes we're making before releasing to Maven Central
* Removed the unit test to focus the user on the integration test and remove any ambiguity about which test works with Contrast
* Added code comments
* Incorporated some feedback from @jennymahmoudi left on Jira

### CODE REVIEW

File: maven-cargo/how-to.md [hunk 1]
```diff
@@ -1,59 +1,59 @@
 # How to Add Contrast to a Maven Project
 
 This how-to walks a Java developer through the necessary steps to add the
-Contrast Java Agent to an existing Maven project and use integration testing to
-find vulnerabilities. It guides you though making changes to an
-[example servlet project](https://github.com/Contrast-Security-OSS/contrast-java-examples/maven-cargo).
-A completed version of the project
-[exists in a git branch](https://github.com/Contrast-Security-OSS/contrast-java-examples/tree/maven-cargo-complete/maven-cargo).
+Contrast Java Agent to an existing Maven project and reuse the project's
+existing integration tests to find vulnerabilities. It guides you though making
```
Comment (jennymahmoudi) [inline maven-cargo/how-to.md]: s/though/through

**Comment by jennymahmoudi** [comment]
Other changes look good. Just that one typo still hanging around.

File: maven-cargo/how-to.md (hunk 1 above)
Comment (gilday) [reply maven-cargo/how-to.md]: fixed

**Review by jennymahmoudi — APPROVED** [review]

**Comment by kaitlynmortimer** [comment]
> I want to move the "how to add integration tests that work with Contrast" content to to an explanation article that is goes into the details of automated testing + Contrast, and we want to keep this how-to focused on Contrast + maven-cargo-plugin.

@gilday If removing this content is best for this set of instructions, I'm fine with that. As far as an article on automated testing is concerned, we'll have to have a separate conversation about that. (I also have a couple bones to pick with the author about this article. We can discuss that later, too.)

**Review by nstoddar — APPROVED** [review]
Looks good to me

**Comment by gilday** [comment]
build is failing due to issue fixed upstream

**Pull request merged**